pages/publish-nft-book/index.vue

```diff
@@ -193,15 +200,18 @@ onMounted(() => {
 
   if (iscnId.value) {
     if (data?.iscnRecord?.iscnId !== iscnId.value) {
-      clearUploadData()
+      clearUploadFileData()
     }
     iscnInputValue.value = iscnId.value as string
     handleIscnSubmit({ iscnId: iscnId.value as string, txHash: '' })
   } else if (classId.value) {
     if (data?.classData?.classId !== classId.value) {
-      clearUploadData()
+      clearUploadFileData()
```

Comment:
`data?. classData ` don't exists in new session format, we can skip this check and clear since it is just session storage.

Reply:
[🎨 Update session handling logic](https://github.com/likecoin/nft-book-press/pull/338/commits/d97d6e2b625fb500a2566d3a44c7e998968e5788)